setup.py

```diff
@@ -19,11 +19,11 @@
 
 setup(
     name='Flask-ICU',
-    version='0.9',
+    version='0.1',
```

Comment:
I think we are further no? Consider what would you expect a `1.0` to contain and then substract `0.1` for each big feature missing. I think we are closer to `0.8` or `0.9` than `0.1` – we at least have an API we believe will be close to `1.0`. Don't sell yourself short!